ulog/log.go

```diff
@@ -186,6 +193,16 @@ func (l *baseLogger) fieldsConversion(keyVals ...interface{}) []zap.Field {
 				fields = append(fields, zap.Marshaler(key, value))
 			case fmt.Stringer:
 				fields = append(fields, zap.Stringer(key, value))
+			case stackTracer:
+				stack := value.StackTrace()
+				lines := make([]zap.Field, 0, len(stack))
+				for _, frame := range stack {
+					s := fmt.Sprintf("%s", frame)
+					l := fmt.Sprintf("line:%d", frame)
```

Comment:
`line` is intentional. Otherwise it can be confusing, what does this magic number mean?
If you think it is obvious, I can remove `line` though